README.md

```diff
@@ -17,3 +17,7 @@ For instructions on how to set up the homework repo, start with the 00-setup.md
 ### Checking for Updates
 
 This repository will be updated throughout the course. Always check to see if there are updates before starting any homework assingment. Make sure to pull any updates (use `git pull <remote> <branch>` to do so).
+
+### Testing a PR
+
+Test test test
```

Comment:
I feel like two is the right number times to write the word 'test' Thanks!

Reply:
Thanks for the feedback, changes made.